Float.TinCan.ActivityLibrary/ActivityLaunchCoordinator.cs

```diff
@@ -128,7 +132,16 @@ public override void Start()
                     downloadStatus.DownloadsCompleted += HandleDownloadCompleted;
                     downloadStatus.DownloadsCancelled += HandleDownloadCancelled;
 
-                    NavigationContext.PresentPage(CreateDownloadStatusPage(downloadStatus));
+                    var statusPage = CreateDownloadStatusPage(downloadStatus);
+                    if (statusPage is PopupPage popupPage)
+                    {
+                        downloadPopupPage = popupPage;
+                        PopupNavigation.Instance.PushAsync(downloadPopupPage, true);
+                    }
+                    else
+                    {
+                        NavigationContext.PresentPage(statusPage);
+                    }
```

Comment:
@EBusch ActivityLibrary doesn't need to care about the type of page -- we can instead do something like this:
```
ShowDownloadStatus(CreateDownloadStatusPage(downloadStatus));
```

And define this method:
```
protected virtual void ShowDownloadStatus(ContentPage downloadStatusPage)
{
  NavigationContext.PresentPage(downloadStatusPage);
}
```

Now the child class can override that method and decide how to show the page.